README.md

```diff
@@ -1,2 +1,106 @@
-# tools
-Tools to assist with integration and deployment of RWAHS software
+# RWAHS Tools
+
+Tools to assist with integration and deployment of RWAHS software.
+
+## Overview
+
+The `rwahs` command handles all steps of integration and deployment via various tasks.
+
+The general syntax is:
+
+    rwahs <task> --env <environment> --component <component>...
+
+Using the shorthand form for parameters:
+
+    rwahs <task> -e <environment> -c <component>...
+
+## Standard Options
+
+* The `--env`/`-e` and `--component`/`-c` parameters apply to most of the tasks.
+* The `--component`/`-c` parameter can be repeated to process multiple components sequentially (in the order specified).
+
+## Environments
+
+The following environments are defined:
+
+* `staging`
+* `uat`
+* `production`
+
+## Components
+
+* `tools` is this tools repository, i.e. a self-update: https://github.com/rwahs/tools
+* `providence` is the CollectiveAccess management application: https://github.com/rwahs/providence
+* `profile` is the CollectiveAccess installation profile (meta-schema): https://github.com/rwahs/installation-profile
+* `config` is the CollectiveAccess local configuration: https://github.com/rwahs/providence-configuration
+* `importer` is the collection of import scripts and queries: https://github.com/rwahs/import-scripts
+
+## Tasks
+
+### Build
+
+Create a new release, which is a tag in the code repository.  
+
+Syntax for the `build` task:
+
+    rwahs build --component <component> [--release <release>]
```

Comment:
I you're using `getopts` you will generally only be able to use `-s` style flags rather than `--long-style` ones.